### ISSUE TITLE
Can't install, error: unused argument (check = True) 

### ISSUE THREAD

**Issue by seantalts** [stan-dev/cmdstanr#112]
**Describe the bug**
Installation in RStudio doesn't seem to work, 

```
✔  checking for file ‘/tmp/Rtmpdhd3hc/remotesaac53502986/stan-dev-cmdstanr-c48a588/DESCRIPTION’ ...
─  preparing ‘cmdstanr’:
✔  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘cmdstanr_0.0.0.9000.tar.gz’
   
Installing package into ‘/home/sean/R/x86_64-pc-linux-gnu-library/3.4’
(as ‘lib’ is unspecified)
* installing *source* package ‘cmdstanr’ ...
** R
** inst
** preparing package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded
Error: package or namespace load failed for ‘cmdstanr’:
 .onLoad failed in loadNamespace() for 'cmdstanr', details:
  call: tempdir(check = TRUE)
  error: unused argument (check = TRUE)
Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/home/sean/R/x86_64-pc-linux-gnu-library/3.4/cmdstanr’
Error: Failed to install 'cmdstanr' from GitHub:
  (converted from warning) installation of package ‘/tmp/Rtmpdhd3hc/fileaac2538d222/cmdstanr_0.0.0.9000.tar.gz’ had non-zero exit status
In addition: Warning message:
In get(var, frame, inherits = FALSE) : internal error -3 in R_decompress1
```
**To Reproduce**
devtools::install_github("stan-dev/cmdstanr")

**Operating system**
Ubuntu 18.04.3 LTS

**Comment by rok-cesnovar** [comment]
Thanks for reporting Sean. 

I tried reproducing this on my laptop running Ubuntu 19.04 but I was unable to. What `devtools` version are you using?

**Comment by rok-cesnovar** [comment]
Ok, scratch that. Its an issue with running R 3.5.0 or lower. But do please confirm that.

See https://github.com/mkearney/pkgverse/issues/3

I am going to apply [this fix](https://github.com/mkearney/pkgverse/blob/d0472b4b6c123af19576912cdbc01deeb8a3f766/R/pkgverse.R#L96), since I dont think there are any other reasons we need to have a `R version > 3.5.0` restriction.

**Comment by rok-cesnovar** [comment]
@seantalts I think its fixed on master, if you have time, please double check and we can close this one. Thanks!

**Comment by seantalts** [comment]
Will do later today, thank you!

**Comment by seantalts** [comment]
This was fixed by #113, thanks!
